recipes-kernel/linux/linux-sunxi_3.4.bb

```diff
@@ -3,10 +3,11 @@ require linux.inc
 DESCRIPTION = "Linux kernel for Allwinner a10/a20 processors"
 
 COMPATIBLE_MACHINE = "(sun4i|sun5i|sun7i)"
+DEPENDS = "sunxi-board-fex"
```

Comment:
Shouldn't this be RDEPENDS, as the results from bulding the sunxi-board-fex recipe aren't needed for the kernel compilation, but rather at runtime.